### PR TITLE
Update dockerfile

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,15 +2,15 @@ FROM golang:1.16 as builder
...
-ARG TERRAGRUNT_VERSION=0.28.8
+ARG TERRAGRUNT_VERSION=0.28.22
 
 # Set Environment Variables
 SHELL ["/bin/bash", "-c"]
 ENV HOME /app
 ENV CGO_ENABLED 0
 
 # Install Packages
-RUN apt-get update -q && apt-get -y install unzip jq
```
Comment (alikhajeh1) [inline Dockerfile:13]: didn't look like we use `jq` in the builder part

File: Dockerfile [hunk 2]
```diff
@@ -45,7 +45,17 @@ RUN chmod +x /app/build/infracost
 # Application
 FROM alpine:3.13 as app
 # Tools needed for running diffs in CI integrations
-RUN apk --no-cache add ca-certificates openssl openssh-client curl git jq
```
Comment (alikhajeh1) [inline Dockerfile:48]: the old diff.sh script used git but not anymore, should we keep git for `scripts/get_version.sh` though?

Comment (aliscott) [reply Dockerfile:48]: `scripts/get_version.sh` is only used by make which we don't include here anyway and don't think we'll need.

**Review by aliscott — APPROVED** [review]

**Comment by aliscott** [comment]
LGTM

**Pull request merged**